test/extensions/transport_sockets/tls/utility_test.cc

```diff
@@ -138,6 +138,34 @@ TEST(UtilityTest, TestGetCertificationExtensionValue) {
   EXPECT_EQ("", Utility::getCertificateExtensionValue(*cert, "foo"));
 }
 
+TEST(UtilityTest, SslErrorDescriptionTest) {
```

Comment:
I don't feel like this test adds much value. Was it needed just to keep coverage up?

Reply:
Yes. It is needed for test coverage.